### PR TITLE
Update zh_tw translation

### PR DESCRIPTION
**Pull request by Evilblue210**

### CODE REVIEW

**Comment by SoLegendary** [comment]
Can you please repoint at 1.19.2-dev?

**Comment by Evilblue210** [comment]
> Can you please repoint at 1.19.2-dev?

Done

File: src/main/resources/assets/reignofnether/lang/zh_tw.json [hunk 1]
```diff
@@ -35,4 +35,669 @@
...
+  "controls.reignofnether.zoom": "Alt + 滑鼠滾輪 - 縮小/放大",
+  "controls.reignofnether.rotate": "滑鼠滾輪 - 旋轉建築物方向",
+
+  "titles.reignofnether.defeated": "您戰敗了",
+  "titles.reignofnether.victorious": "您獲勝了！",
+
+  "fogofwar.reignofnether.warning": "[警告]",
+  "fogofwar.reignofnether.experimental": "您即將為所有玩家啟用戰爭迷霧。該實驗性功能有以下問題：",
+  "fogofwar.reignofnether.optifine_crash": "- 所有安裝 OPTIFINE 的玩家會崩潰",
+  "fogofwar.reignofnether.chunk_bugs": "- 可能會導致區塊渲染錯誤",
```
Comment (notlin4) [inline src/main/resources/assets/reignofnether/lang/zh_tw.json]:
```suggestion
  "fogofwar.reignofnether.chunk_bugs": "- 可能會導致區塊繪製錯誤",
```

File: src/main/resources/assets/reignofnether/lang/zh_tw.json [hunk 2]
```diff
@@ -35,4 +35,669 @@
...
+  "fogofwar.reignofnether.cpu_usage": "- 增加 CPU 使用率（降低區塊渲染距離有緩解）",
```
Comment (notlin4) [inline src/main/resources/assets/reignofnether/lang/zh_tw.json]:
```suggestion
  "fogofwar.reignofnether.cpu_usage": "- 增加 CPU 使用率（降低區塊顯示距離有緩解）",
```

File: src/main/resources/assets/reignofnether/lang/zh_tw.json [hunk 3]
```diff
@@ -35,4 +35,669 @@
...
+  "hud.reignofnether.not_garrisonable": "該建築物無法駐紮",
+  "hud.reignofnether.no_destination": "該傳送門沒有目的地！",
+
+  "server.reignofnether.is_defeated": "%s%s 戰敗了！",
+  "server.reignofnether.lost_buildings": "失去所有建築物",
+  "server.reignofnether.lost_capitol": "%s 失去他們的首府建築，若在 %d 秒前未成功重建，將在地圖上暴露行蹤！",
+  "server.reignofnether.lost_all": "失去所有單位和建築物",
+  "server.reignofnether.players_remaining": "還剩下 %d 名 RTS 玩家",
+  "server.reignofnether.victorious": "%s 獲勝了！",
+  "server.reignofnether.match_reset": "本局已重置！",
```
Comment (notlin4) [inline src/main/resources/assets/reignofnether/lang/zh_tw.json]:
```suggestion
  "server.reignofnether.match_reset": "本局已重設！",
```

File: src/main/resources/assets/reignofnether/lang/zh_tw.json [hunk 4]
```diff
@@ -35,4 +35,669 @@
...
+  "server.reignofnether.match_locked": "本局 RTS 已鎖定，新玩家暫時無法加入。",
+  "server.reignofnether.match_unlocked": "本局 RTS 已解鎖，新玩家可以加入。",
+  "server.reignofnether.sync_enabled": "RTS 登入數據同步已啟用。",
+  "server.reignofnether.sync_disabled": "RTS 登入數據同步已停用。",
```
Comment (notlin4) [inline src/main/resources/assets/reignofnether/lang/zh_tw.json]:
```suggestion
  "server.reignofnether.sync_enabled": "RTS 登入資料同步已啟用。",
  "server.reignofnether.sync_disabled": "RTS 登入資料同步已停用。",
```

File: src/main/resources/assets/reignofnether/lang/zh_tw.json [hunk 5]
```diff
@@ -35,4 +35,669 @@
...
+  "server.reignofnether.bot_added": "%s（電腦）加入遊戲了！",
+  "server.reignofnether.used_cheat": "%s 使用作弊碼：%s %s",
+  "server.reignofnether.enabled_cheat": "%s 啟用作弊 %s",
+  "server.reignofnether.disabled_cheat": "%s 停用作弊 %s",
+  "server.reignofnether.all_cheats": "%s 啟用所有作弊碼",
+  "server.reignofnether.revealed": "%s 未能重建首府，已在地圖上暴露行蹤！",
+
+  "server.resources.reignofnether.not_enough_food": "食物不足",
+  "server.resources.reignofnether.not_enough_wood": "木材不足",
+  "server.resources.reignofnether.not_enough_ore": "礦石不足",
```
Comment (notlin4) [inline src/main/resources/assets/reignofnether/lang/zh_tw.json]:
```suggestion
  "server.resources.reignofnether.not_enough_ore": "礦物不足",
```

File: src/main/resources/assets/reignofnether/lang/zh_tw.json [hunk 6]
```diff
@@ -35,4 +35,669 @@
...
+  "research.reignofnether.strays.tooltip3": "需要墳場。",
+  "research.reignofnether.vindicator_axes": "衛道士斧頭",
+  "research.reignofnether.vindicator_axes.tooltip1": "使衛道士斧頭升級成鑽石斧，提升傷害 +2",
+  "research.reignofnether.wither_clouds": "凋零亡語",
+  "research.reignofnether.wither_clouds.tooltip1": "被凋零骷髏擊殺的敵人",
+  "research.reignofnether.wither_clouds.tooltip2": "死亡後留下有凋零的效果雲。",
+
+  "resources.reignofnether.food": "食物",
+  "resources.reignofnether.wood": "木材",
+  "resources.reignofnether.ore": "礦石",
```
Comment (notlin4) [inline src/main/resources/assets/reignofnether/lang/zh_tw.json]:
```suggestion
  "resources.reignofnether.ore": "礦物",
```

File: src/main/resources/assets/reignofnether/lang/zh_tw.json [hunk 7]
```diff
@@ -35,4 +35,669 @@
...
+  "tutorial.reignofnether.controls": "使用 '/rts-controls' 查看所有控制方式",
+  "tutorial.reignofnether.locked": "本局 RTS 已鎖定。請等待新遊戲開始再加入。",
+  "tutorial.reignofnether.welcome_back": "歡迎回到 Reign of Nether",
+  "tutorial.reignofnether.op_commands": "作為伺服器管理員，您可以使用以下指令：",
+  "tutorial.reignofnether.fog": "/rts-fog enable | disable",
+  "tutorial.reignofnether.lock": "/rts-lock enable | disable",
+  "tutorial.reignofnether.syncing": "/rts-syncing enable | disable",
+  "tutorial.reignofnether.reset": "/rts-reset",
+  "tutorial.reignofnether.sync_disabled1": "RTS 同步已停用，您無法選擇建築物或檢視單位所屬。",
+  "tutorial.reignofnether.sync_disabled2": "要解決這個問題，請聯系伺服器主啟用 RTS 同步並重新登入。",
```
Comment (notlin4) [inline src/main/resources/assets/reignofnether/lang/zh_tw.json]:
```suggestion
  "tutorial.reignofnether.sync_disabled2": "要解決這個問題，請聯絡伺服器主啟用 RTS 同步並重新登入。",
```

File: src/main/resources/assets/reignofnether/lang/zh_tw.json [hunk 8]
```diff
@@ -35,4 +35,669 @@
...
+  "tutorial.reignofnether.make_villager2": "左鍵點擊選擇您的城鎮中心，點擊左下角「村民」按鈕生產一名，之後等待完成。",
+  "tutorial.reignofnether.tip.queue": "提示：您可以生產多個單位加入排程。",
+  "tutorial.reignofnether.nice_work_workers": "做得好。基地的發展少不了這些工人！",
+  "tutorial.reignofnether.tip.cancel": "提示：您可以取消任何在生產中的單位來退還資源，點擊下方單位圖示。",
+  "tutorial.reignofnether.gather_resources": "現在我們有一大群村民，可以開始採集一些資源。",
+  "tutorial.reignofnether.forest": "我們可以從這森林採集木材。",
+  "tutorial.reignofnether.gather_wood": "要採集資源，選擇一名工人，接著右鍵點擊樹。",
+  "tutorial.reignofnether.gather_wood2": "選擇任意村民，接著右鍵點擊任意樹。",
+  "tutorial.reignofnether.well_done": "做得好。",
+  "tutorial.reignofnether.tip.autoreturn": "提示：工人會持續採集直到您命令做其它事。當採集到至少 50 資源時會返回城鎮中心存放資源。",
```
Comment (notlin4) [inline src/main/resources/assets/reignofnether/lang/zh_tw.json]:
```suggestion
  "tutorial.reignofnether.tip.autoreturn": "提示：工人會持續採集直到您命令做其他事。當採集到至少 50 資源時會返回城鎮中心存放資源。",
```

File: src/main/resources/assets/reignofnether/lang/zh_tw.json [hunk 9]
```diff
@@ -35,4 +35,669 @@
...
+  "tutorial.reignofnether.gather_ore": "現在讓我們重複先前操作，來採集礦石吧。",
+  "tutorial.reignofnether.beach_ore": "這裡有片海灘，散布些煤礦和銅礦，我們可以從中獲取。",
+  "tutorial.reignofnether.click_ore": "選擇一名工人，接著右鍵點擊礦方塊。",
+  "tutorial.reignofnether.click_ore2": "選擇任意村民，接著右鍵點擊任意礦方塊。",
+  "tutorial.reignofnether.tip.sand_richer": "提示：沙漠和沙灘等沙地生態域有更多且品質高的礦石。",
+  "tutorial.reignofnether.gather_food": "當您的工人在忙著採集木材和礦石時，讓我們找些食物。",
+  "tutorial.reignofnether.pigs": "這裡有些豬可以打獵並獲得食物。",
+  "tutorial.reignofnether.hunt_pig": "就像採集木材和礦石一樣，選擇一名工人並右鍵點擊一隻動物開始打獵。",
+  "tutorial.reignofnether.hunt_pig2": "選擇任意村民，接著右鍵點擊一隻動物等牠被擊殺。確保村民還有足夠的容量攜帶資源。",
+  "tutorial.reignofnether.empty_pockets": "確保您的工人沒有攜帶其它資源！",
```
Comment (notlin4) [inline src/main/resources/assets/reignofnether/lang/zh_tw.json]:
```suggestion
  "tutorial.reignofnether.empty_pockets": "確保您的工人沒有攜帶其他資源！",
```

File: src/main/resources/assets/reignofnether/lang/zh_tw.json [hunk 10]
```diff
@@ -35,4 +35,669 @@
...
+  "tutorial.reignofnether.forest": "我們可以從這森林採集木材。",
+  "tutorial.reignofnether.gather_wood": "要採集資源，選擇一名工人，接著右鍵點擊樹。",
+  "tutorial.reignofnether.gather_wood2": "選擇任意村民，接著右鍵點擊任意樹。",
+  "tutorial.reignofnether.well_done": "做得好。",
+  "tutorial.reignofnether.tip.autoreturn": "提示：工人會持續採集直到您命令做其它事。當採集到至少 50 資源時會返回城鎮中心存放資源。",
+  "tutorial.reignofnether.gather_ore": "現在讓我們重複先前操作，來採集礦石吧。",
+  "tutorial.reignofnether.beach_ore": "這裡有片海灘，散布些煤礦和銅礦，我們可以從中獲取。",
+  "tutorial.reignofnether.click_ore": "選擇一名工人，接著右鍵點擊礦方塊。",
+  "tutorial.reignofnether.click_ore2": "選擇任意村民，接著右鍵點擊任意礦方塊。",
+  "tutorial.reignofnether.tip.sand_richer": "提示：沙漠和沙灘等沙地生態域有更多且品質高的礦石。",
```
Comment (notlin4) [inline src/main/resources/assets/reignofnether/lang/zh_tw.json]:
```suggestion
  "tutorial.reignofnether.tip.sand_richer": "提示：沙漠和沙灘等沙地生態域擁有更多、更高品質的礦物。",
```

File: src/main/resources/assets/reignofnether/lang/zh_tw.json [hunk 11]
```diff
@@ -35,4 +35,669 @@
...
+  "tutorial.reignofnether.tip.bridges_neutral": "提示：不論是誰建造的橋，都是屬於「中立」。這意思是任何人都可以攻擊、修理和連接新的橋。",
+  "tutorial.reignofnether.tip.crossing_bridges": "提示：過橋時要非常小心。如果單位在行進過程中橋遭到摧毀，會落入水中，移動緩慢且難以反擊！",
+  "tutorial.reignofnether.reinforcements": "援軍已到！現在去摧毀怪物的基地吧！",
+  "tutorial.reignofnether.iron_golem": "援軍中有一隻鐵魔像，現在工人也能建造鐵匠鋪來生產更多鐵魔像。",
+  "tutorial.reignofnether.destroy_monster_base": "準備好您的軍隊並摧毀所有怪物的建築物。",
+  "tutorial.reignofnether.tip.ranged_no_buildings": "提示：大部分遠程單位無法攻擊建築物。命令他們攻擊單位！",
+  "tutorial.reignofnether.tip.monster_capitol": "提示：怪物的首府「陵墓」能產生一定範圍的人造夜晚。如果您在白天將其摧毀，不死怪物將在陽光下燃燒。",
+  "tutorial.reignofnether.congratulations_tutorial": "恭喜！您已完成教學！",
+  "tutorial.reignofnether.continue": "現在解鎖了所有建築物和單位，您可以繼續遊玩。",
+  "tutorial.reignofnether.reset_game": "要重置遊戲並嘗試其他陣營，輸入 /rts-reset",
```
Comment (notlin4) [inline src/main/resources/assets/reignofnether/lang/zh_tw.json]:
```suggestion
  "tutorial.reignofnether.reset_game": "要重設遊戲並嘗試其他陣營，輸入 /rts-reset",
```

File: src/main/resources/assets/reignofnether/lang/zh_tw.json [hunk 12]
```diff
@@ -35,4 +35,669 @@
...
+  "tutorial.reignofnether.move_group": "現在右鍵點擊您想要移動的位置。",
+  "tutorial.reignofnether.move_group2": "圈選好村民後，右鍵點擊地面移動。",
+  "tutorial.reignofnether.great_job": "做得好！",
+  "tutorial.reignofnether.tip.select_same": "提示：您也可以按兩下滑鼠左鍵點擊單位選擇所有相同的單位。",
+  "tutorial.reignofnether.tip.deselect": "提示：如果您想取消選擇單位，按下波浪號（~）鍵",
+  "tutorial.reignofnether.tip.grouping": "提示：如果您是個 RTS 玩家，有可以按下 CTRL + 數字鍵控制編組的功能。",
+  "tutorial.reignofnether.start_base": "現在開始建造基地吧。",
+  "tutorial.reignofnether.capitol": "第一步也是最重要的建築物是首府。",
+  "tutorial.reignofnether.good_spot": "看來這裡是個不錯的地點，夠平坦且空曠又靠近各種資源。",
+  "tutorial.reignofnether.building_resources": "注意建造建築物會消耗資源。還好左上角顯示的木材和礦石資源量足夠我們建造。",
```
Comment (notlin4) [inline src/main/resources/assets/reignofnether/lang/zh_tw.json]:
```suggestion
  "tutorial.reignofnether.building_resources": "注意建造建築物會消耗資源。還好左上角顯示的木材和礦物資源量足夠我們建造。",
```

File: src/main/resources/assets/reignofnether/lang/zh_tw.json [hunk 13]
```diff
@@ -35,4 +35,669 @@
...
+  "tutorial.reignofnether.tip.queue": "提示：您可以生產多個單位加入排程。",
+  "tutorial.reignofnether.nice_work_workers": "做得好。基地的發展少不了這些工人！",
+  "tutorial.reignofnether.tip.cancel": "提示：您可以取消任何在生產中的單位來退還資源，點擊下方單位圖示。",
+  "tutorial.reignofnether.gather_resources": "現在我們有一大群村民，可以開始採集一些資源。",
+  "tutorial.reignofnether.forest": "我們可以從這森林採集木材。",
+  "tutorial.reignofnether.gather_wood": "要採集資源，選擇一名工人，接著右鍵點擊樹。",
+  "tutorial.reignofnether.gather_wood2": "選擇任意村民，接著右鍵點擊任意樹。",
+  "tutorial.reignofnether.well_done": "做得好。",
+  "tutorial.reignofnether.tip.autoreturn": "提示：工人會持續採集直到您命令做其它事。當採集到至少 50 資源時會返回城鎮中心存放資源。",
+  "tutorial.reignofnether.gather_ore": "現在讓我們重複先前操作，來採集礦石吧。",
```
Comment (notlin4) [inline src/main/resources/assets/reignofnether/lang/zh_tw.json]:
```suggestion
  "tutorial.reignofnether.gather_ore": "現在讓我們重複先前操作，來採集礦物吧。",
```

File: src/main/resources/assets/reignofnether/lang/zh_tw.json [hunk 14]
```diff
@@ -35,4 +35,669 @@
...
+  "tutorial.reignofnether.beach_ore": "這裡有片海灘，散布些煤礦和銅礦，我們可以從中獲取。",
+  "tutorial.reignofnether.click_ore": "選擇一名工人，接著右鍵點擊礦方塊。",
+  "tutorial.reignofnether.click_ore2": "選擇任意村民，接著右鍵點擊任意礦方塊。",
+  "tutorial.reignofnether.tip.sand_richer": "提示：沙漠和沙灘等沙地生態域有更多且品質高的礦石。",
+  "tutorial.reignofnether.gather_food": "當您的工人在忙著採集木材和礦石時，讓我們找些食物。",
```
Comment (notlin4) [inline src/main/resources/assets/reignofnether/lang/zh_tw.json]:
```suggestion
  "tutorial.reignofnether.gather_food": "當您的工人在忙著採集木材和礦物時，讓我們找些食物。",
```

File: src/main/resources/assets/reignofnether/lang/zh_tw.json [hunk 15]
```diff
@@ -35,4 +35,669 @@
...
+  "tutorial.reignofnether.pigs": "這裡有些豬可以打獵並獲得食物。",
+  "tutorial.reignofnether.hunt_pig": "就像採集木材和礦石一樣，選擇一名工人並右鍵點擊一隻動物開始打獵。",
```
Comment (notlin4) [inline src/main/resources/assets/reignofnether/lang/zh_tw.json]:
```suggestion
  "tutorial.reignofnether.hunt_pig": "就像採集木材和礦物一樣，選擇一名工人並右鍵點擊一隻動物開始打獵。",
```

File: src/main/resources/assets/reignofnether/lang/zh_tw.json [hunk 16]
```diff
@@ -35,4 +35,669 @@
...
+  "tutorial.reignofnether.gather_resources": "現在我們有一大群村民，可以開始採集一些資源。",
+  "tutorial.reignofnether.forest": "我們可以從這森林採集木材。",
+  "tutorial.reignofnether.gather_wood": "要採集資源，選擇一名工人，接著右鍵點擊樹。",
+  "tutorial.reignofnether.gather_wood2": "選擇任意村民，接著右鍵點擊任意樹。",
+  "tutorial.reignofnether.well_done": "做得好。",
+  "tutorial.reignofnether.tip.autoreturn": "提示：工人會持續採集直到您命令做其它事。當採集到至少 50 資源時會返回城鎮中心存放資源。",
+  "tutorial.reignofnether.gather_ore": "現在讓我們重複先前操作，來採集礦石吧。",
+  "tutorial.reignofnether.beach_ore": "這裡有片海灘，散布些煤礦和銅礦，我們可以從中獲取。",
+  "tutorial.reignofnether.click_ore": "選擇一名工人，接著右鍵點擊礦方塊。",
+  "tutorial.reignofnether.click_ore2": "選擇任意村民，接著右鍵點擊任意礦方塊。",
```
Comment (notlin4) [inline src/main/resources/assets/reignofnether/lang/zh_tw.json]:
```suggestion
  "tutorial.reignofnether.click_ore": "選擇一名工人，接著右鍵點擊礦物方塊。",
  "tutorial.reignofnether.click_ore2": "選擇任何一名村民，接著右鍵點擊任何礦物方塊。",
```

File: src/main/resources/assets/reignofnether/lang/zh_tw.json [hunk 17]
```diff
@@ -35,4 +35,669 @@
...
+  "buildings.piglins.reignofnether.fortress.tooltip1": "一座壯觀的地獄要塞，使軍事傳送門能生產地獄幽靈。",
+  "buildings.piglins.reignofnether.fortress.tooltip2": "最多能駐紮 %d 名單位。",
+  "buildings.piglins.reignofnether.fortress.tooltip3": "需要烈焰聖域和凋零神龕。",
+  "buildings.piglins.reignofnether.hoglin_stables": "豬布獸廄",
+  "buildings.piglins.reignofnether.hoglin_stables.tooltip1": "一座在主世界繁育豬布獸的牧屋。",
+  "buildings.piglins.reignofnether.hoglin_stables.tooltip2": "使軍事傳送門能生產豬布獸。",
+  "buildings.piglins.reignofnether.hoglin_stables.tooltip3": "需要基礎傳送門。",
+  "buildings.piglins.reignofnether.netherwart_farm": "地獄疙瘩農田",
+  "buildings.piglins.reignofnether.netherwart_farm.tooltip1": "\uE001  %d  +  %d（每種植一株作物）",
+  "buildings.piglins.reignofnether.netherwart_farm.tooltip2": "一片地獄疙瘩田，可以種植收獲食物。",
```
Comment (notlin4) [inline src/main/resources/assets/reignofnether/lang/zh_tw.json]:
```suggestion
  "buildings.piglins.reignofnether.netherwart_farm.tooltip2": "一片地獄疙瘩田，可以種植收穫食物。",
```

File: src/main/resources/assets/reignofnether/lang/zh_tw.json [hunk 18]
```diff
@@ -35,4 +35,669 @@
...
+  "buildings.villagers.reignofnether.town_centre.tooltip2": "只能有一座首府建築。",
+  "buildings.villagers.reignofnether.villager_house": "村民房舍",
+  "buildings.villagers.reignofnether.villager_house.tooltip1": "一座簡單屋子，提供人口。",
+  "buildings.villagers.reignofnether.watchtower": "瞭望台",
+  "buildings.villagers.reignofnether.watchtower.tooltip1": "一座塔樓，可以駐紮單位。",
+  "buildings.villagers.reignofnether.watchtower.tooltip2": "駐紮的遠程單位能提升射程。",
+  "buildings.villagers.reignofnether.watchtower.tooltip3": "最多能駐紮 %d 名單位。",
+  "buildings.villagers.reignofnether.wheat_farm": "小麥農田",
+  "buildings.villagers.reignofnether.wheat_farm.tooltip1": "\uE001  %d  +  %d（每種植一株作物）",
+  "buildings.villagers.reignofnether.wheat_farm.tooltip2": "一片小麥田，可以種植收獲食物。",
```
Comment (notlin4) [inline src/main/resources/assets/reignofnether/lang/zh_tw.json]:
```suggestion
  "buildings.villagers.reignofnether.wheat_farm.tooltip2": "一片小麥田，可以種植收穫食物。",
```

File: src/main/resources/assets/reignofnether/lang/zh_tw.json [hunk 19]
```diff
@@ -35,4 +35,669 @@
...
+  "buildings.monsters.reignofnether.laboratory.tooltip1": "一座邪惡實驗室，可基於己建造的其它建築物研究新科技。",
+  "buildings.monsters.reignofnether.laboratory.tooltip2": "",
+  "buildings.monsters.reignofnether.laboratory.tooltip3": "需要墳場",
+  "buildings.monsters.reignofnether.mausoleum": "陵墓（首府）",
+  "buildings.monsters.reignofnether.mausoleum.tooltip1": "一座死亡之墓，能生產殭屍村民。",
+  "buildings.monsters.reignofnether.mausoleum.tooltip2": "使半徑 %d 格範圍內的時間扭曲至午夜。",
+  "buildings.monsters.reignofnether.mausoleum.tooltip3": "只能有一座首府建築。",
+  "buildings.monsters.reignofnether.pumpkin_farm": "南瓜農田",
+  "buildings.monsters.reignofnether.pumpkin_farm.tooltip1": "\uE001  %d  +  %d（每種植一株作物）",
+  "buildings.monsters.reignofnether.pumpkin_farm.tooltip2": "一片南瓜田，可以種植收獲食物。",
```
Comment (notlin4) [inline src/main/resources/assets/reignofnether/lang/zh_tw.json]:
```suggestion
  "buildings.monsters.reignofnether.pumpkin_farm.tooltip2": "一片南瓜田，可以種植收穫食物。",
```

File: src/main/resources/assets/reignofnether/lang/zh_tw.json [hunk 20]
```diff
@@ -35,4 +35,669 @@
...
+  "buildings.monsters.reignofnether.dark_watchtower.tooltip3": "最多能駐紮 %d 名單位",
+  "buildings.monsters.reignofnether.dungeon": "地牢",
+  "buildings.monsters.reignofnether.dungeon.tooltip1": "一座暴怒籠子，有創造苦力怕的能力。",
+  "buildings.monsters.reignofnether.dungeon.tooltip2": "需要實驗室。",
+  "buildings.monsters.reignofnether.graveyard": "墳場",
+  "buildings.monsters.reignofnether.graveyard.tooltip1": "一片死亡之地，能復甦殭屍和骷髏",
+  "buildings.monsters.reignofnether.haunted_house": "鬧鬼房舍",
+  "buildings.monsters.reignofnether.haunted_house.tooltip1": "一座鬼屋，提供人口。",
+  "buildings.monsters.reignofnether.laboratory": "實驗室",
+  "buildings.monsters.reignofnether.laboratory.tooltip1": "一座邪惡實驗室，可基於己建造的其它建築物研究新科技。",
```
Comment (notlin4) [inline src/main/resources/assets/reignofnether/lang/zh_tw.json]:
```suggestion
  "buildings.monsters.reignofnether.laboratory.tooltip1": "一座邪惡實驗室，可基於已建造的其他建築物研究新科技。",
```

File: src/main/resources/assets/reignofnether/lang/zh_tw.json [hunk 21]
```diff
@@ -35,4 +35,669 @@
...
+  "abilities.reignofnether.call_lightning": "召喚閃電",
+  "abilities.reignofnether.call_lightning.tooltip1": "\uE004  %ds  \uE005  %d",
+  "abilities.reignofnether.call_lightning.tooltip2": "在目標位置召喚一道閃電。",
+  "abilities.reignofnether.call_lightning.tooltip3": "可用於充能苦力怕和攻擊敵人。",
+  "abilities.reignofnether.summon_vexes": "召喚惱鬼",
+  "abilities.reignofnether.summon_vexes.tooltip1": "\uE004  %ds",
+  "abilities.reignofnether.summon_vexes.tooltip2": "施法一段時間後召喚 %d 隻惱鬼攻擊喚魔者的目標。",
+  "abilities.reignofnether.summon_vexes.tooltip3": "惱鬼會在 %d 秒後死亡。",
+  "abilities.reignofnether.connect_portal": "連接傳送門",
+  "abilities.reignofnether.connect_portal.tooltip1": "連接到其它您建造的運輸傳送門。",
```
Comment (notlin4) [inline src/main/resources/assets/reignofnether/lang/zh_tw.json]:
```suggestion
  "abilities.reignofnether.connect_portal.tooltip1": "連接到其他您建造的運輸傳送門。",
```

File: src/main/resources/assets/reignofnether/lang/zh_tw.json [hunk 22]
```diff
@@ -35,4 +35,669 @@
...
+  "abilities.reignofnether.sacrifice.in_air": "無法獻祭空中的單位",
+  "abilities.reignofnether.sacrifice.max_spread": "伏聆已達最大範圍",
+
+  "attack_warning.reignofnether.go_to": "前往警告",
+  "attack_warning.reignofnether.ignore": "（右鍵點擊忽略警告）",
+  "attack_warning.reignofnether.danger": "單位正遭受攻擊！",
+
+  "building.reignofnether.outside_map": "距離地圖邊界過近或超出",
+  "building.reignofnether.ground_not_flat": "地面不夠平坦",
+  "building.reignofnether.too_close": "太靠近其它建築物",
```
Comment (notlin4) [inline src/main/resources/assets/reignofnether/lang/zh_tw.json]:
```suggestion
  "building.reignofnether.too_close": "太靠近其他建築物",
```

File: src/main/resources/assets/reignofnether/lang/zh_tw.json [hunk 23]
```diff
@@ -35,4 +35,669 @@
...
+  "buildings.monsters.reignofnether.pumpkin_farm": "南瓜農田",
+  "buildings.monsters.reignofnether.pumpkin_farm.tooltip1": "\uE001  %d  +  %d（每種植一株作物）",
+  "buildings.monsters.reignofnether.pumpkin_farm.tooltip2": "一片南瓜田，可以種植收獲食物。",
+  "buildings.monsters.reignofnether.pumpkin_farm.tooltip3": "南瓜不需要重新種植，但會消耗更多資源。",
+  "buildings.monsters.reignofnether.spider_lair": "蜘蛛巢穴",
+  "buildings.monsters.reignofnether.spider_lair.tooltip1": "一座絲綢洞穴，可以孵化各種蜘蛛。",
+  "buildings.monsters.reignofnether.spider_lair.tooltip2": "需要實驗室。",
+  "buildings.monsters.reignofnether.spruce_bridge": "杉木橋",
+  "buildings.monsters.reignofnether.spruce_bridge.tooltip1": "一座能穿越水域的橋。",
+  "buildings.monsters.reignofnether.spruce_bridge.tooltip2": "必須連接陸地或其它橋。",
```
Comment (notlin4) [inline src/main/resources/assets/reignofnether/lang/zh_tw.json]:
```suggestion
  "buildings.monsters.reignofnether.spruce_bridge.tooltip2": "必須連接陸地或其他橋。",
```

File: src/main/resources/assets/reignofnether/lang/zh_tw.json [hunk 24]
```diff
@@ -35,4 +35,669 @@
...
+  "buildings.monsters.reignofnether.sculk_catalyst.tooltip4": "並提供吸收到的生命值。",
+
+  "buildings.piglins.reignofnether.bastion": "堡壘",
+  "buildings.piglins.reignofnether.bastion.tooltip1": "一座堡壘般的豬布林軍營。",
+  "buildings.piglins.reignofnether.bastion.tooltip2": "使軍事傳送門能生產豬布林軍隊。",
+  "buildings.piglins.reignofnether.bastion.tooltip3": "最多能駐紮 %d 名單位。",
+  "buildings.piglins.reignofnether.bastion.tooltip4": "需要基礎傳送門。",
+  "buildings.piglins.reignofnether.blackstone_bridge": "黑石橋",
+  "buildings.piglins.reignofnether.blackstone_bridge.tooltip1": "一座能穿越水域或熔岩的橋。",
+  "buildings.piglins.reignofnether.blackstone_bridge.tooltip2": "必須連接陸地或其它橋。",
```
Comment (notlin4) [inline src/main/resources/assets/reignofnether/lang/zh_tw.json]:
```suggestion
  "buildings.piglins.reignofnether.blackstone_bridge.tooltip2": "必須連接陸地或其他橋。",
```

File: src/main/resources/assets/reignofnether/lang/zh_tw.json [hunk 25]
```diff
@@ -35,4 +35,669 @@
...
+  "buildings.villagers.reignofnether.iron_golem": "鐵魔像",
+  "buildings.villagers.reignofnether.iron_golem.tooltip1": "可以隨處建造的鐵魔像。",
+  "buildings.villagers.reignofnether.iron_golem.tooltip2": "需要在鐵匠鋪研究科技",
+  "buildings.villagers.reignofnether.library": "圖書館",
+  "buildings.villagers.reignofnether.library.tooltip1": "被書櫃環繞的附魔台。",
+  "buildings.villagers.reignofnether.library.tooltip2": "用於研究魔法相關的升級。",
+  "buildings.villagers.reignofnether.library.tooltip3": "需要奧術塔。",
+  "buildings.villagers.reignofnether.oak_bridge": "橡木橋",
+  "buildings.villagers.reignofnether.oak_bridge.tooltip1": "一座能穿越水域的橋。",
+  "buildings.villagers.reignofnether.oak_bridge.tooltip2": "必須連接陸地或其它橋。",
```
Comment (notlin4) [inline src/main/resources/assets/reignofnether/lang/zh_tw.json]:
```suggestion
  "buildings.villagers.reignofnether.oak_bridge.tooltip2": "必須連接陸地或其他橋。",
```

**Review by notlin4 — CHANGES_REQUESTED** [review]

**Review by SoLegendary — APPROVED** [review]

**Pull request merged**